tests/__init__.py

```diff
@@ -0,0 +1,13 @@
+from twisted.test.proto_helpers import MemoryReactorClock
+from sydent.sydent import Sydent
+
+
+def make_sydent(test_config={}):
```

Comment:
Shouldn't that go to `test_utils`?

Reply:
(or a `utils.py`)

Reply:
We'll add a lot more stuff in the future, so if you want to add to create a util file now it would help for futureproofing.